contrib/internal/httputil/trace_test.go

```diff
@@ -45,6 +45,7 @@ func TestTraceAndServe(t *testing.T) {
 		assert.Equal("service", span.Tag(ext.ServiceName))
 		assert.Equal("resource", span.Tag(ext.ResourceName))
 		assert.Equal("GET", span.Tag(ext.HTTPMethod))
+		assert.Equal(r.Host, span.Tag("http.host"))
```

Comment:
Can you please spell it out using a string literal? Just like the other assertions... (we don't use `r.Method` etc elsewhere either).